downloads.markdown

```diff
@@ -6,156 +6,183 @@ isDownloads: true
 
-*   [Linux](/downloads/linux)
-*   [OS X](https://www.haskell.org/platform/mac.html) (via the core platform)
-*   [Windows](https://www.haskell.org/platform/windows.html) (via the core platform)
+* * *
 
-### Where to get help { #help }
+## Alternative installation options { #ghc-install-manual }
 
-*   For help learning Haskell itself, start with the [Documentation](https://www.haskell.org/documentation) page on the [Haskell Wiki](https://wiki.haskell.org/).
-*   If you need help with [GHC](https://www.haskell.org/ghc)---the Haskell compiler---there is a comprehensive [GHC User Manual](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/index.html).
-*   For help using Cabal to download or create additional packages (see [below](#libraries)), there is the [Cabal User Guide](https://www.haskell.org/cabal/users-guide/).
-*   For help using Stack to download or create packages, see the stack documentation [below](#stackhelp).
-*   Finally, you can ask questions of other Haskell users and experts on the [#haskell IRC channel](irc://irc.freenode.net/haskell) on the Freenode IRC network.
+*   [Using a package manager on linux](#package-manager)
 
-## Stack { #stack }
+*   [Official bindists](#bindists)
 
-### What it is
+*   [Building from source](#from-source)
 
-Stack is a cross-platform build tool for Haskell that handles management of the toolchain (including the GHC compiler and MSYS2 on Windows), building and registering libraries, and more.
+*   [Other options](#other-options)
 
-### What you get
+### Using a package manager on Linux { #package-manager }
 
-*   Once downloaded, it has the capacity to download and install GHC and other core tools.
-*   Project development is isolated within sandboxes, including automatic download of the right version of GHC for a given project.
-*   It manages all Haskell-related dependencies, ensuring reproducible builds.
-*   It fetches from a curated repository of over a thousand packages by default, known to be mutually compatible.
-*   It can optionally use Docker to produce standalone deployments.
+Refer to your distribution package manager documentation. For convenience, below are a few distribution specific instructions, outlining 3rd party repository use as well.
 
-### How to get it
+*Note: Most linux distros don't allow different versions of GHC to be installed in parallel. Sometimes the GHC packages are a bit outdated as well. In either case, consider using [GHCup](#ghcup) instead.*
 
-The [install and upgrade page](http://docs.haskellstack.org/en/stable/install_and_upgrade/) describes how to download Stack on various platforms, although the main three are repeated here:
+*Note: haskell-language-server isn't packaged by most distributions. Instead you will have to install it manually, see the [release page](https://github.com/haskell/haskell-language-server/releases). This is optional.*
```

Comment:
We need to find a better phrasing to convey the information that the VSCode/VSCodium extension takes care of automatically bootstrapping HLS.

Reply:
Done